src/test/kotlin/test/ktortemplate/core/httphandler/TestRoutes.kt

```diff
@@ -310,12 +312,18 @@ class TestRoutes : KoinTest {
         return result
     }
 
-    private suspend fun insertCar(
+    private fun insertCar(
         brand: String = UUID.randomUUID().toString(),
         model: String = UUID.randomUUID().toString()
     ): Car {
-        val newCar = CarSaveCommand(brand, model)
-        return this.carRepository.save(newCar)
+        return dbc.query {
```

Comment:
One of the downsides of not having the repos creating their own transactions is a bit more boilerplate, but I think we can live with that.